locals.tf

```diff
@@ -18,4 +18,10 @@ locals {
       puppet_jenkins_io = "140.211.9.94"
     }
   }
+
+  lets_encrypt_dns_challenged_domains = {
+    "trusted.ci.jenkins.io" = "service_principal"
+    ## TODO: add support for workload identities:
```

Comment:
It is possible to do this with AWS as well (authenticating to Azure)
https://blog.identitydigest.com/azuread-federate-aws/

Reply:
That article is a goldmine! Thanks!

I only wonder if it is worth the time (given the amount of steps to try and automate) compared to the time involved in migrating trusted.ci VM (and associated resources) to Azure instead (and benefit from "standard" Azure Workload identity like we'll implement with cert.ci)?

For the time being, let's start with the SP (which add the burden of the credential to retriueve/insert and rotate)